pkg/sql/catalog/schematelemetry/schema_telemetry_test.go

```diff
@@ -190,4 +190,11 @@ UPDATE system.namespace SET id = %d WHERE id = %d;
 	entries, err := log.FetchEntriesFromFiles(0, math.MaxInt64, 1000, errorRE, log.SelectEditMode(false, false))
```

Comment:
supernit: could we also annotate the arguments to SelectEditMode here

Reply:
done